oidc-conformance-tests/basic/config/service_provider1_config.json

```diff
@@ -34,7 +34,7 @@
         },
         "advancedConfigurations": {
             "discoverableByEndUsers": false,
-            "skipLoginConsent": false
+            "skipLoginConsent": true
```

Comment:
skipping consent as a temporary solution is fine. But we need to investigate and fix this.

Reply:
What is the issue with skipping login consent?

Reply:
It's a issue with the conformance suite. Maybe a bug in the HtmlUnit browser they use. It didn't register redirects from the IS

Reply:
Let's create an issue on both our side and confirmance suite repo to track this